src/micromamba/install.sh

```diff
@@ -1,41 +1,50 @@
 #!/usr/bin/env bash
 
 VERSION=${VERSION:-"latest"}
+# Add destination parameter here?
 
-USERNAME=${USERNAME:-"automatic"}
+# # This looks like it's not needed:
+# USERNAME=${USERNAME:-"automatic"}
 
 set -e
 
+# Is this the standard for dev-container features? It seems like this way each
+# feature will rerun `apt-get update` which seems inefficient. But I suppose
+# this is to prioritize reliability?
 # Clean up
 rm -rf /var/lib/apt/lists/*
 
 if [ "$(id -u)" -ne 0 ]; then
     echo -e 'Script must be run as root. Use sudo, su, or add "USER root" to your Dockerfile before running this script.'
+    # Or it could install at the user level?
```

Comment:
The Dev Container Features are specified in the spec to be run by the root user.